ghost/core/core/server/data/migrations/versions/5.55/2023-07-10-05-16-55-add-built-in-collection-posts.js

```diff
@@ -0,0 +1,99 @@
+const logging = require('@tryghost/logging');
+const {default: ObjectID} = require('bson-objectid');
+const {createTransactionalMigration} = require('../../utils');
+
+const insertPostCollections = async (knex, collectionId, postIds) => {
+    logging.warn(`Batch inserting ${postIds.length} collection posts for collection ${collectionId}`);
+
+    const collectionPosts = postIds.map((postId) => {
+        return {
+            id: (new ObjectID()).toHexString(),
+            collection_id: collectionId,
+            post_id: postId,
+            sort_order: 0
+        };
+    });
+
+    await knex.batchInsert('collections_posts', collectionPosts, 1000);
+};
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Populating built-in collections');
+
+        const existingLatestCollection = await knex('collections')
+            .where({
+                slug: 'latest'
+            })
+            .first();
+
+        if (!existingLatestCollection) {
+            logging.warn('Latest collection does not exists, skipping');
+        } else {
+            const latestPostsRows = await knex('posts')
+                .select('id')
+                .where({
+                    type: 'post'
+                });
+
+            const latestPostsIds = latestPostsRows.map(row => row.id);
+
+            await insertPostCollections(knex, existingLatestCollection.id, latestPostsIds);
+        }
+
+        const existingFeaturedCollection = await knex('collections')
+            .where({
+                slug: 'featured'
+            })
+            .first();
+
+        if (!existingFeaturedCollection) {
+            logging.warn('Featured collection does not exist, skipping');
+        } else {
+            const featuredPostsRows = await knex('posts')
+                .select('id')
+                .where({
+                    featured: true,
```

Comment:
Does this need a filter for `status: 'published'`?

Reply:
same answer as above for "latest" collection.